pkg/databaseclaim/databaseclaim.go

```diff
@@ -1216,27 +1220,32 @@ func (r *DatabaseClaimReconciler) manageUserAndExtensions(ctx context.Context, r
 		}
 	}
 
-	if status.UserUpdatedAt == nil || time.Since(status.UserUpdatedAt.Time) > rotationTime {
-		logger.V(1).Info("rotating_users", "rotationTime", rotationTime)
+	if statusNewDB.UserUpdatedAt == nil || time.Since(statusActiveDB.UserUpdatedAt.Time) >= rotationTime {
+		logger.V(1).Info("rotating user password",
```

Comment:
querying logs without spaces is faster
```suggestion
		logger.V(1).Info("rotating_user_password",
```